1.0/docs/devguide/styling.md

```diff
@@ -699,3 +699,21 @@ that includes a `custom-style` element.
       } 
     </style>      
 
+## Third-party libraries that modify local DOM
+
+If you are using a third-party library that adds local DOM nodes to your
+Polymer element, you may notice that styles on the element do not update
+properly. 
+
+The correct way to add DOM nodes to a Polymer element's local DOM is via
+`Polymer.dom`. This method adds the proper scoping classes
+to the element, which ensures that styles are updated properly. 
+
+Many libaries do not know about this method, so they do not add DOM nodes 
+correctly. Call `scopeSubtree` in `ready` to enable a mutation observer on 
+your element's local DOM. This mutation observer detects changes to the 
+local DOM, and applies proper CSS scoping to the element and its descendants.
+
+    ready: function() {
+      this.scopeSubtree(this.$.content, true);
```

Comment:
:heavy_check_mark: I actually thought that this was referring to the content tag and agree that it could have created a rabbit hole